### PR TITLE
Feature description

### PR DESCRIPTION
**Pull request by rsanchez-dv**
Added description component.
Some parts need to still be wired up to the back end but the overall structure is there.

### CODE REVIEW

**Review by mattfreshwaters — APPROVED** [review]
lgtm

File: src/components/DescriptionBox/Description.vue [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    props: ['listingId','listing'],
+    data(){
+        return{
+          'data': ''
+        }
+    },
+    created(){
+       axios({
+              method: 'GET',
+              url: `http://localhost:8080/api/description/${this.listingId}`,
```
Comment (Eliotdoesprogramming) [inline src/components/DescriptionBox/Description.vue:110]: description should be pulled from http request on the parent component and then passed down as a prop

Edit: appears we have some updates to be made in the backend to properly map the relationship

**Review by Eliotdoesprogramming — CHANGES_REQUESTED** [review]
only needs the one change in the front end, but we'll have to make sure the back end is properly supplying data

File: src/components/DescriptionBox/Description.vue (hunk 1 above)
Comment (rsanchez-dv) [reply src/components/DescriptionBox/Description.vue:110]: Ah ok. Ill update that.

Comment (rsanchez-dv) [reply src/components/DescriptionBox/Description.vue]: K, updated.

**Pull request merged**